app/helpers/action_links_helper.rb

```diff
@@ -22,6 +22,23 @@ def renew_link_for(resource)
     ad_privacy_policy_path(reg_identifier: resource.reg_identifier)
   end
 
+  def display_renewal_link_for?(resource)
+    return false unless WasteCarriersEngine::FeatureToggle.active?(:renewal_reminders)
+
+    display_renew_link_for?(resource)
+  end
+
+  def renewal_link_for(resource)
```

Comment:
Might be something for a different PR, but as this URL is now being used here and in the letters and emails, would be great if we were just generating it in a single place instead :D

Reply:
Wholeheartedly agree! But I wasn't sure how to go about it. I rely on stuff in the action links helper here which isn't needed in the renewal reminder mailer.

So I couldn't see how I could split it out into something shared, without a wholesale refactoring. That felt too risky at this time. But I would _love_ it if there was some solution I've overlooked or I'm just overthinking what is needed 🤷‍♂️ 😁

Reply:
Ignore me, being dumb. Got a simple solution for this 🤦